app/src/main/java/an/maguste/android/navier/data/Movie.kt

```diff
@@ -1,20 +1,52 @@
 package an.maguste.android.navier.data
 
+import an.maguste.android.navier.BuildConfig
 import android.os.Parcelable
 import kotlinx.parcelize.Parcelize
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
 
 @Parcelize
+@Serializable
 data class Movie(
+        @SerialName("id")
         val id: Int,
+        @SerialName("title")
         val title: String,
+        @SerialName("overview")
         val overview: String,
+        @SerialName("poster_path")
         val poster: String,
+        @SerialName("backdrop_path")
         val backdrop: String,
+        @SerialName("vote_average")
         val ratings: Float,
+        @SerialName("adult")
         val adult: Boolean,
-        val runtime: Int,
+        @SerialName("runtime")
+        val runtime: Int? = null,
+        @SerialName("vote_count")
         val reviews: Int,
-        val genres: List<Genre>,
-        val actors: List<Actor>,
-        val like: Boolean = false
-) : Parcelable
\ No newline at end of file
+        @SerialName("genre_ids")
+        val genreIds: List<Int>? = listOf(),
```

Comment:
Так как ты используешь значение по умолчанию, используй not null тип `List<Int>`
```suggestion
        val genreIds: List<Int> = listOf(),
```

Reply:
В твоем объекте появились "дубликаты" genreIds и genres, потому что этот класс пытается делать несколько вещей. С одной стороны, он как Dto (data transfer object) получает данные по сети, с другой стороны, как модель предоставляет данные пользователю. 

Обрати внимание на то, как было реализовано в файле JsonLoad.kt Было две сущности: одна - для парсинга из Json, вторая - модель данных. B данные маппились из одной структуры в другую. 

Нужно сделать два объекта: MovieDto (со списком идентификаторов жанров) и Movie (со списком жанров текстом).